shared/src/main/scala/coop/rchain/shared/MapOps.scala

```diff
@@ -0,0 +1,8 @@
+package coop.rchain.shared
+
+object MapOps {
+  def merge[K, V1, V2](map1: Map[K, V1], map2: Map[K, V2], v1: V1, v2: V2): Map[K, (V1, V2)] =
+    (map1.keySet ++ map2.keySet).map { i =>
```

Comment:
some property tests for this function would be nice to have

Reply:
Done